lib/index.js

```diff
@@ -1,6 +1,6 @@
-var stream = require('./stream');
-var es = require('event-stream');
-var _ = require('lodash');
+var stream = require("./stream");
+var es = require("event-stream");
+var _ = require("lodash");
```

Comment:
Fwiw, these changes are not really in line with our common coding style, even if it’s not reported as a linting error currently

Reply:
Ahh sorry - I think it was caused automatically when I saved in VS-code. Would you like me to fix and maybe add a `.vscode` setting with preference for `'` quotes - or maybe even adding a `prettier.config.js`if you have one from other projects that would fit?

Reply:
Ahh ok. Seems adding this to `package.json` makes eslint run: `"lint": "eslint lib test examples --fix"`. Do you wan't me to add that in a new commit?

Reply:
sure, whichever way is easiest for you, as long as it works in the end :)

Reply:
Ok - I fixed the linting issues and added a `lint` script to fix it in the future.

Reply:
Is there something else I should add before the PR can be merged?

Reply:
Is there something else I should add before the PR can be merged?